README.md

```diff
@@ -1,2 +1,73 @@
 # RoverSim
 Open Projects 2021
+
+## Abstract
+
+The project aims to introduce the Path-Planning phase to a wireless rover to make it capable to navigate autonomously in an environment with the help of path planning algorithms (Dijkstra and SLAM) parsed by a python script. It aims to reach the destination through the shortest path and shortest time. This was done using ROS navigation stack by performing a simulation of a rover model in Gazebo environment, by running ROS (Melodic Morenia) in Ubuntu 18.04. The robot used in this project is Turtlebot 3.
+
+![gazebo_sim](https://github.com/brahatesh/RoverSim/blob/main/Images%20and%20Videos/Gifs/gazebo_sim.gif "Simulation in Gazebo")
+
+![rviz_sim](https://github.com/brahatesh/RoverSim/blob/main/Images%20and%20Videos/Gifs/rviz_sim.gif "Simulation in RViz")
```

Comment:
Center align all the images using the `<p>` paragraph tag and `align` attribute. For eg:
`<p align="center">`
  `<img src="https://github.com........." alt="LED Matrix">`
  `<i>LED Matrix</i>`
`</p>`

Refer this README for the same: https://raw.githubusercontent.com/marsiitr/Rube-Goldberg-Machine/master/README.md

Also resize a few images by assigning the dimensions. And include these 2 images before abstract as mentioned in the guidelines.